Files/ViewModels/ItemViewModel.cs

```diff
@@ -1242,14 +1245,23 @@ private async Task EnumFromStorageFolderAsync()
                 }
                 if (count % 300 == 0)
```

Comment:
Maybe do the same as standard enumeration: `count == 32 || count % 300 == 0` so the user starts seeing items sooner